### DISCUSSION TITLE
External window title for a process (MacOS)

### DISCUSSION THREAD

**Discussion by JohnKesko** [AvaloniaUI/Avalonia#10564] [Q&A] [answered]
I'm new to Avalonia and I am trying to get an external window handle outside my application, for example chrome.
I have it working on windows using process.MainWindowTitle.Contains("mywebsite") => // do stuff with the handle.

Is it possible for Mac with Avalonia?

**Comment by maxkatz6** [comment, marked as answer]
I don't think it's related to Avalonia at all.
Process API is .NET BCL API, so you can ask this question at "dotnet/runtime" repository.
If .NET API doesn't support it on macOS, you can search for macOS native APIs (in objective c or swift) and rewrite that code in c#.

**Reply by JohnKesko** [reply]
I see, thanks for reply! Yeah, found out about calling the runtime 'objc_callsomething' stuff so will continue with that.
